Yank/pipeline.py

```diff
@@ -491,7 +491,7 @@ def read_system_files(positions_file_path, parameters_file_path, system_options,
 
     # Store numpy positions and create SamplerState.
     positions = positions_file.getPositions(asNumpy=True)
-    sampler_state = mmtools.states.SamplerState(positions=positions)
+    sampler_state = mmtools.states.SamplerState(positions=positions, box_vectors=box_vectors)
```

Comment:
Oh, shoot, I hadn't realized that since `ReplicaExchange` had been written in such a way that the storage layer *required* `box_vectors` to be present.

Reply:
It really should not, but the sampler writes them anyways. I don't remember without digging into the code what we do to the `sampler_states` stored in the `Repex` object so that their `box_vectors` are not `None`, even in implicit/vacuum solvents.